ft_appenddata.m

```diff
@@ -336,6 +322,31 @@
   data.label = data.label(reorder);
 end
 
+removesens = 0;
+if haselec || hasgrad || hasopto
+  sens = cell(1, Ndata);
+  for j=1:Ndata
+    if haselec, sens{j} = varargin{j}.elec; end
+    if hasgrad, sens{j} = varargin{j}.grad; end
+    if hasopto, sens{j} = varargin{j}.opto; end
+  end
+  for j=1:Ndata
+    if j>1
+      if ~isequaln(sens{j}, sens{1})
+        removesens = 1;
+        warning('sensor information does not seem to be consistent across the input arguments');
+        break;
+      end
+    end
+  end
+  if strcmp(cfg.appendsens, 'yes')
+    fprintf('concatenating sensor information across all input arguments\n');
```

Comment:
i.e. the output can have both a grad and an elec if I do appendsens=yes with EEG as input 1 and MEG as input 2.